HIP/hip-1064.md

```diff
@@ -0,0 +1,163 @@
+---
+hip: 1064
+title: Daily Rewards For Active Nodes
+author: Neeharika Sompalli <@Neeharika-Sompalli>
+working-group: Leemon Baird (@lbaird), Richard Bair <@rbair23>, Michael Tinker <@tinker-michaelj>
+requested-by: Hashgraph
+type: Standards Track
+category: Core
+needs-tsc-approval: Yes
+status: Draft
+last-call-date-time: 2024-04-08T07:00:00Z
+created: 2024-10-17
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/1063
+updated: 2024-10-17
+---
+
+## Abstract
+
+This HIP proposes a reward mechanism that will incentivize nodes to remain active on the network. The definition of active node is described below. The minimum reward per day for nodes will be a configurable property decided by the governance of the deployed network. This minimum reward may be adjusted based on fees already paid throughout the day.
+
+## Motivation
+
+The goal is to give a strong incentive for node operators to keep them active and properly configured throughout every staking period.
+
+## Rationale
+
+Clearly defining the criteria for node activity, we can ensure that rewards are distributed fairly, incentivizing nodes to remain active and contribute to the network’s stability and growth.
+
+### Terminology
+
+In the hashgraph algorithm, honest nodes that are online and contributing to consensus will regularly create unique famous witnesses, called judges, in nearly every round.
+
+This HIP defines a node as *active* if it creates a judge in a significant fraction of rounds during a staking period, for example 80%. Any node that is trustworthy and submits events will create judges. If the node doesn’t submit events or when other nodes don’t build on this node’s events ,  the node will not create judges.
+
+**User Stories**
+
+The implementation will be tested under various scenarios, including:
+
+- As a node operator, I will expect to get a minimum node reward when the node is active for at least `nodes.activeRoundsPercent`  rounds
+- As a node operator, I will not expect to get any node rewards when the node is not active for at least `nodes.activeRoundsPercent`  rounds
+- As a network operator, when `nodes.adjustNodeFees` is set to true, each nodes gets reward reduced by the average node fees already collected during that period.
+- As a network operator, when `nodes.adjustNodeFees` is set to false, node rewards are not reduced by the average node fees already collected during that period.
+- As  a network operator, each active node will get minimum reward defined by `nodes.minNodeReward`
+
+## Specification
+
+There will be three configurable properties added, which can be changed using `FileUpdate` transaction signed by the governance of the deployed network.
```

Comment:
I think we should not mention how a property is changed. This may change in the future.